src/apps/onboarding/src/components/InputTextAutoSave/index.tsx

```diff
@@ -37,6 +37,7 @@ const InputTextAutoSave: FC<InputTextProps> = (props: InputTextProps) => {
         <InputText
             {...props}
             value={value}
+            forceUpdateValue
```

Comment:
fogot to mention this, make sure that whatever uses InputTextAutoSave is not broken by this (shouldn't, but just to be safe).